packages/cli/src/config/settingsSchema.ts

```diff
@@ -38,12 +38,12 @@ export type SettingsValue =
  * (e.g. an enum or true/false) rather than allowing for free form input
  * (like a number or string).
  */
-export const TOGGLE_TYPES: ReadonlySet<SettingsType | undefined> = new Set([
+export const TOGGLE_TYPES: ReadonlySet<SettingsType | undefined> = new Set<SettingsType | undefined>([
```

Comment:
OPTIONAL NIT: This specific declaration is unnecessary (the ReadonlySet structural type enforces this).  This could be removed, but I don't feel strongly at all.